### PR TITLE
Correct upstream API References property casing, bump velero docs version

### LINKED ISSUE #770: Bug: API_Ref incorrect casing on some fields
### Contact Details

_No response_

### Describe bug

Some fields are incorrectly PaschalCase instead of expected camelCase user would use in the DPA Spec.

### What happened?

A bug happened!

### OADP Version

1.0.3 (Stable)

### OpenShift Version

_No response_

### Velero pod logs

_No response_

### Restic pod logs

_No response_

### Operator pod logs

_No response_

### New issue

- [X] This issue is new

Related to https://issues.redhat.com/browse/OADP-642

### PR DESCRIPTION
**Pull request by kaovilai**
Corrects API references property fields casing to match what user would type into dpa spec.

Hyperlink to types on pkg.go.dev

Using pseudo version pkg.go.dev to include datamover features property.
https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220722142130-a02487302a02/api/v1alpha1#Features
May follow up by replacing pseudo-version with tagged version 1.1.0

This PR is accurate as of https://github.com/openshift/oadp-operator/commit/a02487302a02ce9cb014c17a8fe1072c7ae4a805

Do this to get psuedo version of latest master commit
```
go get -d github.com/openshift/oadp-operator/api@master
go: github.com/openshift/oadp-operator/api@master: github.com/openshift/oadp-operator/api@v0.0.0-20220722142130-a02487302a02: invalid version: missing github.com/openshift/oadp-operator/api/go.mod at revision a02487302a02
```

- [x] Todo: Update link after https://github.com/openshift/oadp-operator/pull/747

Closes #770 #768

### CODE REVIEW

File: docs/API_ref.md [hunk 1]
```diff
@@ -1,78 +1,79 @@
...
+| podConfig                       | *[PodConfig](https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220722142130-a02487302a02/api/v1alpha1#PodConfig)              | Velero Pod specific configuration                                                                                                                                                                                                                        |
+| logLevel                       | [string](https://pkg.go.dev/builtin#string)              | Velero server’s log level (default info, use debug for the most logging). Valid options are error, warn, warning, info, debug, trace                                                                                                                                                                                                                        |
```
Comment (RichardHoch) [inline docs/API_ref.md:48]: "warn" and "warning" ?

File: docs/API_ref.md [hunk 2]
```diff
@@ -1,78 +1,79 @@
...
 
 | Property | Type                                                                                                | Description                                                                                    |
 |----------|-----------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------|
-| name     | metav1. ObjectMeta                                                                                  |                                                                                                |
+| name     | [metav1.ObjectMeta](https://pkg.go.dev/k8s.io/apimachinery/pkg/apis/meta/v1#ObjectMeta)                                                                                  |                                                                                                |
 | velero   | [*velero.VolumeSnapshotLocationSpec](https://velero.io/docs/v1.6/api-types/volumesnapshotlocation/) | Location to store volume snapshots. For further details, see  [here] ( config/bsl_and_vsl.md). |
```
Comment (RichardHoch) [inline docs/API_ref.md:29]: version 1.6? Should be version 1.9.

File: docs/API_ref.md (hunk 1 above)
Comment (kaovilai) [reply docs/API_ref.md:48]: Thanks. `trace;debug;info;warning;error;fatal;panic` are valid options

File: docs/API_ref.md [hunk 3]
```diff
@@ -1,70 +1,77 @@
...
-| tolerations         | [[]corev1. Toleration](https://pkg.go.dev/k8s.io/api/core/v1#Toleration)                  | Tolerations defines the list of tolerations to be applied to Velero Deployment/Restic daemonset                                                             |
+| nodeSelector        | map [ [string](https://pkg.go.dev/builtin#string) ] [string](https://pkg.go.dev/builtin#string)                                                                     | NodeSelector defines the nodeSelector to be supplied to Velero/Restic podSpec                                                             |
+| tolerations         | [][corev1.Toleration](https://pkg.go.dev/k8s.io/api/core/v1#Toleration)                  | Tolerations defines the list of tolerations to be applied to Velero Deployment/Restic daemonset                                                             |
 | resourceAllocations | [corev1.ResourceRequirements](https://pkg.go.dev/k8s.io/api/core/v1#ResourceRequirements) | Set specific resource  `limits`  and  `requests`  for the Velero/Restic pods. For more information, go  [here] ( config/resource_req_limits.md). |
```
Comment (RichardHoch) [inline docs/API_ref.md:73]: Should this type include the parameter "labels"?

Comment (kaovilai) [reply docs/API_ref.md:74]: Thanks!

**Review by sseago — APPROVED** [review]

**Comment by kaovilai** [comment]
switching to v0.0.0-20220727202820-c5f87df8fafd for links

File: docs/API_ref.md [hunk 4]
```diff
@@ -1,70 +1,78 @@
...
 
-### Features
+### [Features](https://pkg.go.dev/github.com/openshift/oadp-operator@v0.0.0-20220727202820-c5f87df8fafd/api/v1alpha1#Features)
```
Comment (shubham-pampattiwar) [inline docs/API_ref.md:75]: Can you please update the dataMover API as well ? Ref: https://github.com/openshift/oadp-operator/blob/master/bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml#L446

**Review by shubham-pampattiwar — APPROVED** [review]

**Review by savitharaghunathan — COMMENTED** [review]
/lgtm

**Pull request merged**